dashboard/test/fixtures/regional_partner_mappings.csv

```diff
@@ -0,0 +1,6 @@
+Region
+12345
+WA
```

Comment:
Maybe more of a programs team question, but are we ok with partners having a mix of zips and states? Or should they all be one or the other?

Reply:
I think we are ok with it generally, I'm not sure if there's a case of a partner covering multiple states but that's a case I could think of--where they cover part of 1 state and all of another?

Reply:
Yep you're right just looked at our existing mappings, looks like we have a few who have both.